test/container/simple_test.exs

```diff
@@ -1,12 +0,0 @@
-defmodule TestContainer.Container.SimpleTest do
```

Comment:
the system has begun to limit it self to not allow running the Container struct directly. Another struct must be defined and it must implement ContainerBuilder. This is too much work to fix this test